statsfs/otelstats/subsys_metric_parser.go

```diff
@@ -0,0 +1,199 @@
+package otelstats
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
```

Comment:
Generally we prefer the following structure when struct is involved.

```
type A struct {
   ...
}

func (a *A) f1(..) {
}

func (a *A) f2(..) {
}

type B struct {
}

...
```

This provides a natural flow when reading the code.